src/python/grpcio/grpc/_channel.py

```diff
@@ -85,7 +85,7 @@ def _deadline(timeout):
 
 
 def _unknown_code_details(unknown_cygrpc_code, details):
-  return b'Server sent unknown code {} and details "{}"'.format(
+  return 'Server sent unknown code {} and details "{}"'.format(
```

Comment:
The cython function ```str_to_bytes``` is used any time a str/bytes object is passed to the cython level, which ascii encodes any non-byte strings.